cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -629,8 +634,20 @@ and then restart CDAP.
 
      for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
      
-If you are upgrading a secure Hadoop cluster, see the notes above about
-:ref:`configuring secure Hadoop <install-secure-hadoop>`. 
+#. You should recompile and redeploy your application. This will allow you to see your old
```

Comment:
Edited it as per comment, and moved to end as an additional step, as it didn't really follow with the other text.